browser/lowbrow/menu.js

```diff
@@ -0,0 +1,66 @@
+const modifier = if (process.platform == "darwin") {
```

Comment:
Awesome you can assign the result of an if statement in Javascript.  Big feature from Haskell I miss in other languages, notably Python.